Cant Compile

On kiss linux w/ musl.

getting this; 
```
/usr/bin/ld: /tmp/cccHhIDF.o:(.bss+0x0): multiple definition of `signal_map'; /tmp/ccHEIdfG.o:(.data.rel.local+0x0): first defined here
collect2: error: ld returned 1 exit status
make: *** [Makefile:8: hummingbird] Error 1
```

--- a/src/signal.h
+++ b/src/signal.h
@@ -6,7 +6,7 @@
 #define SIGUNUSED SIGSYS
 #endif
 
-int (*signal_map[SIGUNUSED])(void);
+extern int (*signal_map[SIGUNUSED])(void);
 
 int restart_machine(void);
 int poweroff_machine(void);

